edenai_apis/apis/groq/groq_api.py

```
@@ -0,0 +1,172 @@
+    ChatMessageDataClass as ChatMultimodalMessageDataClass,
+)
+from edenai_apis.features.multimodal.chat.chat_dataclass import (
+    StreamChat as StreamChatMultimodal,
+)
+from edenai_apis.features.text.chat.chat_dataclass import ChatDataClass, StreamChat
+from edenai_apis.llmengine.llm_engine import LLMEngine
+from edenai_apis.loaders.data_loader import ProviderDataEnum
+from edenai_apis.loaders.loaders import load_provider
+from edenai_apis.utils.types import ResponseType
+from edenai_apis.features.llm.llm_interface import LlmInterface
+from edenai_apis.features.llm.chat.chat_dataclass import ChatDataClass
```

Ruff (0.8.2)

Redefinition of unused `ChatDataClass` from line 16

(F811)